about/css/style.css

```diff
@@ -0,0 +1,183 @@
+/* top */
+
+.top-wrapper{
+    padding-top: 215px;
+    padding-bottom: 196px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.top-logo{
+    margin-bottom: 30px;
+}
+
+.top-content{
+    font-family: 'Libre Baskerville', serif;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 30px;
+    max-width: 370px;
+    text-align: center;
+}
+
+/* articles */
+
+.articles{
+    background: #1C1C1C;
+    color: #ffffff;
+    padding-bottom: 139px;
+    position: relative;
+}
+
+.article-list::after{
+    content: url(/about/images/v-line.svg);
+    position: absolute;
+    bottom: -5px;
+    left: 160px;
+}
+
+.article-list::before{
+    content: url(/about/images/v-line.svg);
+    position: absolute;
+    top: 0px;
+    right: 160px;
+}

```

Comment:
At this case better to avoid use separate resource file. Because here it's just a white line and we can make it in pure CSS right here.
Moreover we should care about height of this element. In case content inside parent block will be small - line should have adjusted height. So better to use relative units as percents.